addons/crm/tests/common.py

```diff
@@ -219,6 +219,18 @@ def setUpClass(cls):
             'country_id': cls.env.ref('base.us').id,
             'zip': '97648',
         })
+        cls.contact_without_email = cls.env['res.partner'].create({
+            'name': 'Jhon Doe',
+            'email': False,
+            'mobile': cls.test_phone_data[1],
+            'phone': cls.test_phone_data[2],
```

Comment:
It bothers me that they share a phone number with another contact

Maybe change the address to be unique too